CalcualtorLogic.js

```diff
@@ -28,6 +28,9 @@ class Calculator {
     constructor(selector) {
         this.hostElem = document.querySelector(selector);
     };
-    render() {};
+    render() {
+        const wrapperDiv = document.createElement('div');
+        wrapperDiv.classList.add = 'wrapper';
```

Comment:
This is incorrect.
`.add` is function.
![image](https://user-images.githubusercontent.com/16171515/108407875-3b089100-721c-11eb-9094-71882a091012.png)